Oracle/harvest_local.ctl

```diff
@@ -0,0 +1,28 @@
+options (errors=0, skip=1)
```

Comment:
ugh. this is a lot of ceremony to create a table and insert one row.

Let's use a little python ditty that calls sqlplus.